src/pmdas/linux/help

```diff
@@ -775,6 +775,7 @@ e.g. mirroring the results from existing performance tools:
 @ kernel.all.sysfork fork rate metric from /proc/stat
 @ kernel.all.running number of currently running processes from /proc/stat
 @ kernel.all.blocked number of currently blocked processes from /proc/stat
+@ kernel.all.btime boot time from /proc/stat
```

Comment:
@shogo-matsumoto could we name this metric 'kernel.all.boottime' instead?

Reply:
@natoscott Yes, of course.